presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -83,6 +83,10 @@ PeriodicTaskManager::PeriodicTaskManager(
       server_(server) {}
 
 void PeriodicTaskManager::start() {
+  numDriverThreads_ = size_t(
```

Comment:
Could be an accessor on `PrestoServer` so we don't repeat the logic here

Reply:
@Yuhta 
The code allows `server_` (`PrestoServer*`) to be null, so can't really use this for bullet-proof code.
I also thought just taking it from `driverCPUExecutor_` itself, but that could be null too.
So, doing this. :)

Reply:
@Yuhta 

I can do
`x = driverCPUExecutor_ ? driverCPUExecutor_->threads() : size_t();
`
If that's better?